### PR TITLE
Mhv 60168 content updates

### PR DESCRIPTION
**Pull request by robertroddev**
## Summary
Added feature flag for allergies content
Added link to landing and list page to take user to '/my-health/medical-records/allergies' if the feature flag is turned on

## Related issue(s)
[Change content and add link for allergies](https://jira.devops.va.gov/browse/MHV-60168)
image

## Testing done
Unit test to cover both true and false states of feature flag

ran coverage report
image

image
image

## Screenshots

|         | flag on | flag off |
| ------- | ------ | ----- |
| landing page  |image|image|
| list page |image|image|

## What areas of the site does it impact?

mhv medications

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

**Review by vmenshutin-bylight — DISMISSED** [review]

File: src/applications/mhv-medications/containers/Prescriptions.jsx [hunk 1]
```diff
@@ -472,10 +476,21 @@ const Prescriptions = () => {
...
+            {!showAllergiesContent && (
+              <span>
+                If you print or download this list, we’ll include a list of your
+                allergies.
+              </span>
+            )}
           </p>
+          {showAllergiesContent && (
+            <div className="vads-u-margin-bottom--3">
+              <a href="/my-health/medical-records/allergies" rel="noreferrer">
```
Comment (agravell047) [inline src/applications/mhv-medications/containers/Prescriptions.jsx:489]: Would it be possible to use just an <a> tag and apply the styles directly? Just for semantics.

File: src/applications/mhv-medications/containers/Prescriptions.jsx [hunk 2]
```diff
@@ -472,10 +476,21 @@ const Prescriptions = () => {
             data-testid="Title-Notes"
           >
             When you share your medications list with providers, make sure you
-            also tell them about your allergies and reactions to medications. If
-            you print or download this list, we’ll include a list of your
-            allergies.
+            also tell them about your allergies and reactions to medications.{' '}
+            {!showAllergiesContent && (
+              <span>
+                If you print or download this list, we’ll include a list of your
```
Comment (agravell047) [inline src/applications/mhv-medications/containers/Prescriptions.jsx:482]: For navigating with assistive-tech, its usually better to use a React fragment here to render all this text contained inside the <p> tag.

File: src/applications/mhv-medications/containers/Prescriptions.jsx (hunk 1 above)
Comment (robertbylight) [reply src/applications/mhv-medications/containers/Prescriptions.jsx:489]: let me try 👍

Comment (robertbylight) [reply src/applications/mhv-medications/containers/Prescriptions.jsx:489]: tried to add the `className="vads-u-margin-bottom--3"` to the `<a>` tag directly but it didn't apply the style.

File: src/applications/mhv-medications/containers/Prescriptions.jsx (hunk 2 above)
Comment (robertbylight) [reply src/applications/mhv-medications/containers/Prescriptions.jsx:482]: yeah good idea! will do

**Review by Andrew565 — DISMISSED** [review]
Approved but you should look into the failing unit tests to make sure there's nothing important there.

**Review by vmenshutin-bylight — APPROVED** [review]

**Review by Andrew565 — APPROVED** [review]

**Pull request merged**